src/plugins/components/PluginsList/PluginsList.tsx

```diff
@@ -72,6 +72,7 @@ const PluginList: React.FC<PluginListProps> = props => {
           plugin =>
             plugin ? (
               <TableRow
+                data-test="id"
```

Comment:
test-id should be more precise
```suggestion
                data-test-id="plugin"
```